addons/web/static/tests/legacy/views/kanban_tests.js

```diff
@@ -2997,6 +3081,81 @@ QUnit.module('Views', {
         kanban.destroy();
     });
 
+    QUnit.test('drag and drop record if grouped by date/time field with attribute allow_drag_drop_create: true', async function (assert) {
+        assert.expect(14);
+
+        this.data.partner.records[0].date = '2017-01-08';
+        this.data.partner.records[1].date = '2017-01-09';
+        this.data.partner.records[2].date = '2017-02-08';
+        this.data.partner.records[3].date = '2017-02-10';
+        this.data.partner.records[0].datetime = '2017-01-08 10:55:05';
+        this.data.partner.records[1].datetime = '2017-01-09 11:31:10';
+        this.data.partner.records[2].datetime = '2017-02-08 09:20:25';
+        this.data.partner.records[3].datetime = '2017-02-10 08:05:51';
+
+        var kanban = await createView({
+            View: KanbanView,
+            model: 'partner',
+            data: this.data,
+            arch: '<kanban class="o_kanban_test">' +
```

Comment:
This one is mostly an advice for you to write tests more efficiently (because it's a test and we don't really care about formatting conventions here anyway): you can use template litterals to write multiline xml templates: 
```js
arch: `
    <kanban class="o_kanban_test">
        <field name="foo" />
    </kanban>
`,